cardano-shell.cabal

```diff
@@ -118,6 +118,7 @@ executable node-ipc
       base >=4.7 && <5
     , cardano-shell
     , cardano-prelude
+    , safe-exceptions
```

Comment:
Why?

Reply:
should use everywhere functions from `Control.Exception.Safe`?

Reply:
The purpose of [`Control.Exception.Safe`](https://hackage.haskell.org/package/safe-exceptions) is to avoid some abuses of the `SomeException` type (like catching catching all exceptions and either silently swallowing them, and/or retrying the the function that threw it).

In my opinion this is a poor alternative to:
* Handling all exceptions correctly.
* Avoiding the use of exceptions for control flow and error handing (ie using `Either` and `ExceptT` instead).
* Catching exceptions thrown by upstream libraries and converting them to error types.